composer.json

```diff
@@ -89,6 +89,12 @@
         ],
         "post-update-cmd": [
             "@auto-scripts"
+        ],
+        "cs": "vendor/bin/php-cs-fixer fix --config=.php-cs-fixer.dist.php -v --dry-run --stop-on-violation --diff --using-cache=no",
```

Comment:
I know this is from before, but I think we should use cache, so that we can speed up our GitHub Action a bit

Reply:
371d48ca7789b95ebccf2150259c36406b19f6b3 @sweoggy